Library/Homebrew/test/rubocops/lines_cop_spec.rb

```diff
@@ -419,7 +419,7 @@ class Foo < Formula
       end
     end
 
-    it "with ENV.universal_binary" do
+    it "with ENV.universal_binary 2" do
```

Comment:
Why `2`? Also please create full sentences when using `it`, or use `specify` instead of `it`.

Reply:
Tried replacing with somewhat better phrases.